package.json

```diff
@@ -33,7 +34,7 @@
   "scripts": {
     "build": "rm -rf lib && babel src --out-dir lib",
     "docs-build": "rm -rf docs && esdoc -c esdoc.json",
-    "lint": "eslint ./",
+    "lint": "eslint --fix ./",
```

Comment:
Do not change the scripts, it's out of scope

Reply:
Fixed